modules/setup/library/Setup/WebWizard.php

```diff
@@ -568,11 +568,10 @@ public function getRequirements($skipModules = false)
         $set = new RequirementSet();
 
         $set->add(new PhpVersionRequirement(array(
-            'condition'     => array('>=', '5.3.2'),
+            'condition'     => array('>=', '5.4'),
```

Comment:
@lippserd Aren't we using 5.6 as our new requirement?

Reply:
Yes, will be in a new PR. Did not want to mix things here.

Reply:
@nilmerg I agree with you (and w/ @lippserd as well), but I wouldn't like all of our customers to jump out of the window at the same time. So I'd do it in more than one step.